packages/binaryen/binaryen.0.8.1/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+synopsis: "OCaml bindings for Binaryen"
+maintainer: "oscar@grain-lang.org"
+authors: "Oscar Spencer"
+license: " Apache-2.0"
+homepage: "https://github.com/grain-lang/binaryen.ml"
+bug-reports: "https://github.com/grain-lang/binaryen.ml/issues"
+depends: [
+  "ocaml" {>= "4.09"}
+  "dune" {>= "2.7.1"}
```

Comment:
The dune-project file requires dune 2.6. I am fine to merge as is, but if you prefer I can lower this to 2.6 before merging

Reply:
@mseri I was under the impression that the `dune` version and `dune lang` version were separate and that we could upgrade dune without upgrading the language in our dune files?

Reply:
Yes, you can. What I meant is that if you are using dune lang version 2.6, the package is automatically compatible with dune 2.6, even if you use newer versions of dune

Reply:
I see. We actually had dune pinned at 2.6.1 for a long time so this project would build with esy and then we were finally able to upgrade to >=2.7.1. Maybe it was 2.7 that didn't work (?) so we didn't want to change to >=2.6 and instead upgraded to >=2.7.1. This change was also applied to the opam file in order to keep those dependencies in sync.

Reply:
```suggestion
  "dune" {>= "2.6"}
```
I leave it up to you. I will merge tomorrow, if you decide you want this change feel free to accept the suggestion, otherwise I merge as is